engine/classes/ElggUser.php

```diff
@@ -590,7 +590,13 @@ public function getFriendsObjects($options = array(), $limit = 10, $offset = 0)
 	 */
 	public function countObjects($subtype = "") {
 		elgg_deprecated_notice("\ElggUser::countObjects() is deprecated. Use elgg_get_entities()", 1.9);
-		return count_user_objects($this->getGUID(), $subtype);
+		$options = [
+			'count' => true,
+		];
+		if ($subtype) {
+			$options['subtype'] = $subtype;
+		}
+		return (int)$this->getObjects($options);
```

Comment:
These are rarely used. I wanted to fix in the simplest way since we don't have any tests.